src/hotspot/cpu/x86/macroAssembler_x86.hpp

```diff
@@ -955,6 +955,74 @@ class MacroAssembler: public Assembler {
   // contained in the location described by entry (not the address of entry)
   void jump(ArrayAddress entry, Register rscratch);
 
+  // Adding more natural conditional jump instructions
+  void ALWAYSINLINE jo(Label& L, bool maybe_short = true) { jcc(Assembler::overflow, L, maybe_short); }
+  void ALWAYSINLINE jno(Label& L, bool maybe_short = true) { jcc(Assembler::noOverflow, L, maybe_short); }
+  void ALWAYSINLINE js(Label& L, bool maybe_short = true) { jcc(Assembler::positive, L, maybe_short); }
```

Comment:
Isn't js -> jump is sign flag is set -> Assembler::negative?
Correspondingly jns, js_b, jns_b should also be corrected.